frontend/public/module/k8s/index.ts

```diff
@@ -242,8 +242,17 @@ export type PodStatus = {
   [key: string]: any;
 };
 
+export type PodMetadata ={
```

Comment:
We already have a common metadata definition. No need to redefine it.

https://github.com/openshift/console/blob/master/frontend/public/module/k8s/index.ts#L33-L41